### ISSUE TITLE
class balanced loss?

### ISSUE THREAD

**Issue by 2533245542** [CoinCheung/pytorch-loss#34]
Ever considered implementing class balanced loss? I found your repo useful and would like to see it include more.

https://github.com/vandit15/Class-balanced-loss-pytorch

**Comment by CoinCheung** [comment]
Sorry for being so late. I was busy with other things when this issue is created. Frankly, I do not have such a plan to add it because the implementation you posted above is good and well documented. 

I tried it in sementic segmentation task of cityscapes, but observed a bit drop of miou. The model I used is deeplabv3+, and I simply replace cross entropy with this loss without other changings. Maybe I should be more careful to tune the training parameters.
